adoc/ii-Strategy/SA.adoc

```diff
@@ -7,59 +7,45 @@ The strategy elements are typically used to model the strategic direction and ch
 ////
 
-One key example here is that you no longer need to plan the hardware sizing for the next 5 years, as a larger, faster, or even smaller infrastructure is only one reboot away. 
+One key benefit of the cloud is that a business no longer needs to plan hardware sizing for the next five or more years.  Larger, faster, or even smaller infrastructure is only one reboot away.
 
-"Rightsizing" or infrastructure optimization remains an important task and should be front of mind when managing an SAP Landscape in the cloud. Not taking care of this cloud benefit could have large cost implications, companies that “rightsize” their workloads are often able to cut costs by of 30-60%.
+However, "rightsizing" (or infrastructure optimization) remains an important consideration.  Businesses that actively manage "rightsizing" their environment can cut operating costs by 30 to 60 percent.
 
 //image::SA-Strategy.png[title="Solution Architecture - {useCase} Strategy", scaledwidth=80%]
 
 
 === Context
 
-There are many benefits gained when moving your SAP workloads to the cloud:
+There are many benefits gained when moving SAP workloads to the cloud:
 
-Quick deployment:: If you need fast application implementation and deployment, the cloud is the best choice. You can set up a cloud environment within a few hours, whereas, in-house IT infrastructure set-up takes days and sometimes months to order it and set up. With the cloud, IT teams can easily configure and manage the setup remotely.
-+    
-This is the area where SUSE's automation solution for SAP can help.
+Quick deployment:: If you need fast application implementation and deployment, the cloud is the best choice. You can set up a cloud environment within a few hours, whereas, in-house IT infrastructure can take days or even  months to order, install, configure, and bring online.  With the cloud and SUSE's automation solution for SAP, IT teams can easily configure and manage the setup remotely.
 
-Reduce Costs:: Many businesses spent a lot of their capital on maintaining their IT Infrastructure. With help of the cloud, they can transform their CapEx (Capital Expenditure) to stable OpEx (Operating Expenditure).
-Costs can be controlled as the cloud model is a ‘pay as you use’ scenario and many cloud vendors provide several consumption options to provide more choice to their customers. 
-+    
-This is an area where SUSE can help together with the cloud providers to offer the right options, but this also means you need to adapt to how you use the resources and the SAP software.
+Reduce Costs:: Many businesses experience large CapEx (Capital Expenditure) to maintain IT infrastructure, with purchases often in advance of actual need.  SUSE, together with the cloud providers, help businesses transform to a more efficient OpEx (Operating Expenditure) model, with ‘pay as you use’ options that empower businesses with greater agility.
```

Comment:
pls. add the other to bullet points again - its too simplified and does not show that you need to do something on the applicaton (SAP) side too - if you want to reduce costs!

Reply:
I'll merge in an update to this effect.